homeassistant/components/camera/arlo.py

```diff
@@ -81,6 +104,28 @@ def name(self):
         return self._name
 
     @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return merge_two_dicts(super().state_attributes, {
```

Comment:
The entity base class will take care of merging state_attributes with device_state_attributes. Just return a new dict with the device_state_attributes here.

Reply:
Make sure there's no I/O going on when you get the attributes of the camera.

Reply:
Looking at the source library, it appears these attributes are calculated immediately, so nothing I’m doing should be introducing I/O.

https://github.com/tchellomello/python-arlo/blob/master/pyarlo/camera.py